README.md

```diff
@@ -143,6 +143,7 @@ library:
 | `build-type` | · | `Simple`, or `Custom` if `custom-setup` exists | Must be `Simple`, `Configure`, `Make`, or `Custom` | | |
 | `extra-source-files` | · | | Accepts [glob patterns](#file-globbing) | | |
 | `extra-doc-files` | · | | Accepts [glob patterns](#file-globbing) | | `0.21.2` |
+| `extra-files` | · | | Accepts [glob patterns](#file-globbing) | | `0.39.0` |
```

Comment:
```suggestion
| `extra-files` | · | | Accepts [glob patterns](#file-globbing) | | `0.38.1` |
```

Reply:
The reason I've held off on the Hpack version is that there may be other changes soon that could be released together (and `extra-files` on its own is not a burning platform) - I'm currently (off repository) looking at moving #518 forward.

Reply:
I say "release early, release often" and "minimize dependencies between tasks".

Reply:
Noted, but in this case #518 is on the heals of this one - and maybe #605 too, if I can convince you of its utility.